adapters/adform/adform.go

```diff
@@ -0,0 +1,380 @@
+package adform
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/pbs"
+	"golang.org/x/net/context/ctxhttp"
+)
+
+type AdformAdapter struct {
+	http         *adapters.HTTPAdapter
+	URI          string
+	usersyncInfo *pbs.UsersyncInfo
```

Comment:
Now that we have a usersync module, does the adapter still need usersync info?

Reply:
No need, have removed it